### PR TITLE
Force send Enable field for LogConfig

### PR DESCRIPTION
**Pull request by skmatti**
/assign @rramkumar1

### CODE REVIEW

File: pkg/composite/gen.go [hunk 1]
```diff
@@ -3220,6 +3220,9 @@ func (backendService *BackendService) ToGA() (*compute.BackendService, error) {
 	if ga.Iap != nil {
 		ga.Iap.ForceSendFields = []string{"Enabled", "Oauth2ClientId", "Oauth2ClientSecret"}
 	}
+	if ga.LogConfig != nil {
```
Comment (rramkumar1) [inline pkg/composite/gen.go:3223]: I don't think you can modify gen.go? I think you have to modify the code which generates this file.

Comment (skmatti) [reply pkg/composite/gen.go:3229]: Got it. Modified the source of the generate file and updated the generated code

**Pull request merged**